### ISSUE TITLE
The size of WSI-VQA datasets

### ISSUE THREAD

**Issue by Lewislou** [cpystan/WSI-VQA#2]
The released WSI-VQA dataset only has 7139 VQA pairs. Why the size of the dataset is not the same as the reported in the paper? 
Btw, I have checked the slide names in [TCGA ](https://portal.gdc.cancer.gov/analysis_page?app=Downloads). The overall number of slides of TCGA-BRCA is 1016+2563. And I can only find 541 slides in TCGA-BRCA correspond to the case ids in the WsiVQA.json. Can you provide the slide ids that you used to create the WSI-VQA dataset? 

![image](https://github.com/user-attachments/assets/edb80f2d-4aa5-491d-ae04-a2528a77722a)

**Comment by cpystan** [comment]
The WsiVQA.json only contains the training set which contains 7139 pairs. The validation set contains 798 pairs and test set contains 735 pairs. We will upload the validation set and test set soon.
In terms of the ids, we use the 'DX' slide as the input. The total number of slides is nearly 1000.

**Comment by cpystan** [comment]
The ids we use are included in dataset/splits_0.csv

**Comment by Lewislou** [comment]
Thanks for your quick response, my questions are well solved~

**Comment by Lewislou** [comment]
Hi, I have noticed in datasets.py. Some WSIs in test set or val set in splits_0.csv are filtered if they have the same patient id with train set? So in testing stage, only 86 WSIs but not 98 WSIs in splits_0.csv are evaluated?
![image](https://github.com/user-attachments/assets/b0015e97-11ab-4f14-ade5-2a1ef8c669d0)

**Comment by cpystan** [comment]
Yes. You are right.
